src/src/PluginStructs/P116_data_struct.cpp

```diff
@@ -145,6 +151,16 @@ bool P116_data_struct::plugin_init(struct EventStruct *event) {
         }
         break;
       }
+      case ST77xx_type_e::ST7796s_320x480: // ST7789vw 135x240
```

Comment:
Comment doesn't match the enum.
And to be honest, the enum is rather descriptive, so the comment is not needed.